test/output/slim/inline_anchor.html

```diff
@@ -0,0 +1,43 @@
+<!-- .basic -->
+<a class="bare" href="http://www.asciidoctor.org">http://www.asciidoctor.org</a>
+
+<!-- .basic_with_text -->
+<a href="irc://irc.freenode.org/#asciidoctor">Asciidoctor IRC channel</a>
+
+<!-- .basic_with_target_blank -->
+<a href="view-source:asciidoctor.org" target="_blank">Asciidoctor homepage</a>
+
+<!-- .basic_with_role -->
+<a class="green" href="http://discuss.asciidoctor.org/"><strong>mailing list</strong></a>
+
+<!-- .xref -->
+
+<!-- .xref_with_text -->
+The section <a href="#page-break">Page break</a> describes how to add a page break.
+
+<!-- .xref_resolved_text -->
+Refer to <a href="#_section_a">Section A</a>.
+
+<!-- .bibref -->
+
+<!-- .ref -->
+
+<!-- .xref-interdoc -->
+
+<!-- .xref-with-text -->
+
+<!-- .xref-resolved-text -->
+
+<!-- .xref-reftext -->
+
+<!-- .xref-xrefstyle -->
+
+<!-- .bibref-with-text -->
+
+<!-- .link -->
+
+<!-- .link-with-text -->
+
+<!-- .link-with-target-blank -->
+
+<!-- .link-with-role -->
```

Comment:
Hm, it seems that you’ve somehow mixed examples generated using older DocTest version and the latest. It’d be better/easier to remove all output examples generated from reference examples and generate them from scratch.